src/ducks/apps/components/Sections/components/AppsSection.spec.jsx

```diff
@@ -44,6 +44,6 @@ describe('AppsSection component', () => {
       .dive() // shallow on more level on first app item
     appItem.simulate('click')
     expect(mockOnAppClick.mock.calls.length).toBe(1)
-    expect(mockOnAppClick.mock.calls[0][0]).toBe(mockApps[0].slug)
+    expect(mockOnAppClick.mock.calls[0][0]).toBe('konnector-bouilligue')
```

Comment:
Why not use the mock object directly? It avoids hard written strings twice in tests

Reply:
Because the apps are now sorted, it is not necessarily the first one.

Do you think it would be clearer with 

```
     expect(mockOnAppClick.mock.calls[0][0]).toBe(mockApps[6].slug)  // IDK the real index of konnector-bouilligue but it is not 0
```

?

Reply:
Ok I see, I understand now, let's keep it like thi.